homeassistant/components/lg_thinq/sensor.py

```diff
@@ -255,9 +261,93 @@
+    ),
+}
+TIME_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    TimerProperty.LIGHT_START: SensorEntityDescription(
+        key=TimerProperty.LIGHT_START,
+        translation_key=TimerProperty.LIGHT_START,
+    ),
+    TimerProperty.ABSOLUTE_TO_START: SensorEntityDescription(
+        key=TimerProperty.ABSOLUTE_TO_START,
+        translation_key=TimerProperty.ABSOLUTE_TO_START,
+    ),
+    TimerProperty.ABSOLUTE_TO_STOP: SensorEntityDescription(
+        key=TimerProperty.ABSOLUTE_TO_STOP,
+        translation_key=TimerProperty.ABSOLUTE_TO_STOP,
+    ),
+    TimerProperty.TARGET: SensorEntityDescription(
+        key=TimerProperty.TARGET,
+        translation_key=TimerProperty.TARGET,
+    ),
+}
+TIMER_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    TimerProperty.RELATIVE_TO_START: SensorEntityDescription(
+        key=TimerProperty.RELATIVE_TO_START,
+        device_class=SensorDeviceClass.DURATION,
+        native_unit_of_measurement=UnitOfTime.MINUTES,
+        translation_key=TimerProperty.RELATIVE_TO_START,
+    ),
+    TimerProperty.RELATIVE_TO_START_WM: SensorEntityDescription(
+        key=TimerProperty.RELATIVE_TO_START,
+        device_class=SensorDeviceClass.DURATION,
+        native_unit_of_measurement=UnitOfTime.MINUTES,
+        translation_key=TimerProperty.RELATIVE_TO_START_WM,
+    ),
+    TimerProperty.RELATIVE_TO_STOP: SensorEntityDescription(
+        key=TimerProperty.RELATIVE_TO_STOP,
+        device_class=SensorDeviceClass.DURATION,
+        native_unit_of_measurement=UnitOfTime.MINUTES,
+        translation_key=TimerProperty.RELATIVE_TO_STOP,
+    ),
+    TimerProperty.RELATIVE_TO_STOP_WM: SensorEntityDescription(
+        key=TimerProperty.RELATIVE_TO_STOP,
+        device_class=SensorDeviceClass.DURATION,
+        native_unit_of_measurement=UnitOfTime.MINUTES,
+        translation_key=TimerProperty.RELATIVE_TO_STOP_WM,
+    ),
+    TimerProperty.SLEEP_TIMER_RELATIVE_TO_STOP: SensorEntityDescription(
+        key=TimerProperty.SLEEP_TIMER_RELATIVE_TO_STOP,
+        device_class=SensorDeviceClass.DURATION,
+        native_unit_of_measurement=UnitOfTime.MINUTES,
+        translation_key=TimerProperty.SLEEP_TIMER_RELATIVE_TO_STOP,
+    ),
+    TimerProperty.REMAIN: SensorEntityDescription(
+        key=TimerProperty.REMAIN,
+        device_class=SensorDeviceClass.DURATION,
+        native_unit_of_measurement=UnitOfTime.MINUTES,
+        translation_key=TimerProperty.REMAIN,
+    ),
+    TimerProperty.RUNNING: SensorEntityDescription(
+        key=TimerProperty.RUNNING,
+        device_class=SensorDeviceClass.DURATION,
+        native_unit_of_measurement=UnitOfTime.MINUTES,
+        translation_key=TimerProperty.RUNNING,
+    ),
+    TimerProperty.TOTAL: SensorEntityDescription(
+        key=TimerProperty.TOTAL,
+        device_class=SensorDeviceClass.DURATION,
+        native_unit_of_measurement=UnitOfTime.MINUTES,
+        translation_key=TimerProperty.TOTAL,
+    ),
```

Comment:
How often do these states change? Because if this changes 1 minute every minute, we still rather want to keep a timestamp of when we expect it to happen.